### ISSUE TITLE
"value is undefined" error with nested field and not defined initial form values

### ISSUE THREAD

**Issue by TeChn4K** [Lemoncode/fonk-required-by-field-validator#2]
My form have nested fields and `requiredByField` validators.

When my initial form values are not defined, I get this error : `TypeError: "value is undefined"`

Here is a fix:

Replace 
https://github.com/Lemoncode/fonk-required-by-field-validator/blob/0684947b8b08c8dc3e8ec1ff20b4bd35d6464a5a/src/validator.ts#L38
By 
```js
 field.split('.').reduce((value, key) => value ? value[key] : null, values); 
```

**Comment by TeChn4K** [comment]
Repro : https://codesandbox.io/s/falling-forest-jupl3

Remove requiredByField validation or the dot in field name and it will work

**Comment by TeChn4K** [comment]
See my PR #3

**Comment by brauliodiez** [comment]
Thanks we will review it as soon as possible, in order to merge it

**Comment by nasdan** [comment]
Hi @TeChn4K , we have just released a new [release version 1.0.1](https://www.npmjs.com/package/@lemoncode/fonk-required-by-field-validator/v/1.0.1) including the fix for the nested fields. Could you try it?.
Thank you.

**Comment by TeChn4K** [comment]
It works well!
Thank you for your work, Fonk is a very good library!
